src/gc/impl/conservative/gc.d

```diff
@@ -43,6 +43,7 @@ import core.gc.gcinterface;
 import rt.util.container.treap;
```

Comment:
I think you should define a version (e.g. `COLLECT_FORKED`) that is set only for supporting platforms. Most code should only be enabled with that version.

Reply:
Or use `static if (HaveFork)`.